tox.ini

```diff
@@ -6,13 +6,13 @@ skipsdist = True
 [testenv]
 setenv =
   VIRTUAL_ENV={envdir}
-  PYTHONWARNINGS=default::DeprecationWarning,ignore::DeprecationWarning:distutils,ignore::DeprecationWarning:site
+  PYTHONWARNINGS=default::DeprecationWarning,ignore:The distutils package is deprecated,ignore:The distutils.sysconfig module is deprecated,ignore:the imp module is deprecated,ignore:SelectableGroups dict interface is deprecated
```

Comment:
Updated the list of ignored warnings, this hasn't been touched since the original @pasuder's commit in 2018 😅